database/postgresconn/postgresconn.go

```diff
@@ -310,7 +324,9 @@ func (p *Postgres) SetVersion(version int, dirty bool) error {
 }
 
 func (p *Postgres) Version() (version int, dirty bool, err error) {
-	query := `SELECT version, dirty FROM ` + pq.QuoteIdentifier(p.config.migrationsSchemaName) + `.` + pq.QuoteIdentifier(p.config.migrationsTableName) + ` LIMIT 1`
+	query := fmt.Sprintf(`SELECT version, dirty FROM %q.%q`+
```

Comment:
probably a future pr, but i'm curious if you can seperate the database access concerns from the business logic of running a migration

Reply:
maybe one day, all of the tests run in docker container, so the testability wouldn't be supstantially higher, feels like busy work.

Reply:
@AndrewWinterman the migation is actaully run by other types, "driver" and "migrate", postgresconn.go is the database parts of the migration